src/ast.cpp

```diff
@@ -2226,4 +2226,20 @@ namespace Sass {
     }
   }
 
+  //////////////////////////////////////////////////////////////////////////////////////////
+  // Convert map to (key, value) list.
+  //////////////////////////////////////////////////////////////////////////////////////////
+  List* Map::to_list(Context& ctx, ParserState& pstate) {
```

Comment:
This might be some what wasteful but it's equivalent to what Ruby Sass does and makes list functions easy to reason about for newcomers.